### ISSUE TITLE
Choosing the value of alpha and beta

### ISSUE THREAD

**Issue by deepikay** [xiaohuiyan/BTM#15]
In the research paper, it is mentioned that alpha = 0.05 for short text collections and =50/K for normal text collection and beta = 0.01. Should we proceed with the default values mentioned in the runExample.sh?

**Comment by xiaohuiyan** [comment]
Yes, this parameter setting is fine in most of data I tested. However, the
best values of these parameters depend on data.

2017-10-31 14:01 GMT+08:00 deepikay <notifications@github.com>:
